code/modules/clothing/head/misc_special.dm

```diff
@@ -213,3 +214,25 @@
 	icon_state = "foilhat"
 	item_state = "paper"
 	siemens_coefficient = 2
+
+/obj/item/clothing/head/celtic
+	name = "\improper Celtic crown"
+	desc = "According to legend, Celtic kings would use crowns like this one to shield their subjects from harsh winters back on Earth."
+	icon_state = "celtic_crown"
+	wizard_garb = 1
+
+/obj/item/clothing/head/celtic/equipped(mob/living/carbon/human/H, head)
+	if(istype(H) && head == src)
+		H.species.cold_level_1 = -1
+		H.species.cold_level_2 = -1
+		H.species.cold_level_3 = -1
+		H.species.can_be_hypothermic = 0
+		H.faction = "frost"
```

Comment:
This shouldn't be an issue in this PR because there's no need to rework the entire system for this but I reckon factions should probably be made defines later down the line